packages/typespec-ts/static/static-helpers/pagingHelpers.ts

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
These codes are till in our emitter repo, as files I think we still need these headers.

Reply:
emitter will add these two line from https://github.com/kazrael2119/autorest.typescript/blob/b61ee35905401ede30f1597ce49b410dd4a4d732/packages/typespec-ts/src/utils/emitUtil.ts#L61-L65

Reply:
@lirenhe do you know the guidance here? do we need to add license headers for these static resources? these codes are files not under `src` folder but `static` [folder ](https://github.com/Azure/autorest.typescript/tree/main/packages/typespec-ts/static). And we would copy these files into publish artifacts as static resources also.

FYI - the generated code would always have lisence header.

Reply:
![image](https://github.com/user-attachments/assets/b8ad2e42-c4d0-4e00-bfe4-79fecee0fe68)